src/main/resources/templates/mdBenefitsFlow/householdMoneyOnHand.html

```diff
@@ -22,27 +21,12 @@
                 content=~{::moneyOnHand})}">
                 <th:block th:ref="moneyOnHand">
                   <th:block
-                      th:replace="~{fragments/inputs/checkboxInSet :: checkboxInSet(inputName='moneyOnHandTypes',value=#{moneyonhand-types.option1}, label=#{moneyonhand-types.option1})}"/>
+                      th:each="option : ${T(org.mdbenefits.app.data.enums.MoneyOnHandType).values()}">
+                    <th:block
+                        th:replace="~{fragments/inputs/checkboxInSet :: checkboxInSet(inputName='moneyOnHandTypes',value=${option.getValue()}, label=#{${option.getLabelSrc()}})}"/>
```

Comment:
This looks great - I think there are a few variables in the inputs file that can come out, because the second page has been deleted

Reply:
We don't really need to use the enum's `value` field here (we may not even need the enum to have `value`).  You can just use `option.name()` - which will give you the enum name (SAVINGS, CHECKING, etc).   It's easy to look up then when filling in the pdf.  And it's a consistent value.

Reply:
I am down for using .name() but want to keep value in for now, as I plan to use that to fill out the PDF